pyciemss/integration_utils/intervention_builder.py

```diff
@@ -0,0 +1,43 @@
+from typing import Dict, List
+
+import torch
+from chirho.interventional.ops import Intervention
+
+
+def param_value_objective(
```

Comment:
Can you add a return type here?

Reply:
Should be `Callable[[torch.Tensor], Dict[float, Dict[str, Intervention]]]`

Reply:
Done